charts/hono/values.yaml

```diff
@@ -157,6 +182,12 @@ dataGridExample:
   # imageName contains the name (including tag)
   # of the container image to use for the example data grid.
   imageName: infinispan/server:11.0.7.Final-1
+  # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+  # configuration property of the component's liveness probe.
+  livenessProbeInitialDelaySeconds: 600
```

Comment:
Using 600s here for infinispan seems quite high, considering that the previous value was 50s.